streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -612,25 +582,4 @@ public void shouldFailOnEosAbortTxFatal() {
 
         assertThat(thrown.getMessage(), equalTo("KABOOM!"));
     }
-
-    @Test
-    public void shouldFailOnCloseFatal() {
```

Comment:
We do not created new test classes of ActiveTaskCreator / StandbyTaskCreator, but we should still have those coverage to make sure the exception thrown from producers are wrapped correctly. 

Also it seems in the new code we no long rethrow -- is that intentional. I left a comment above.

Reply:
Good catch. Actually, there's a bunch of coverage missing from TaskManager. I'll add several more tests.